src/featureFlags.js

```diff
@@ -99,4 +110,28 @@ export class FeatureFlags extends events.EventEmitter {
   static async clearIgnoreWalletServiceFlag() {
     await localStorage.removeItem(IGNORE_WALLET_SERVICE_FLAG);
   }
+
+  /**
+   * Uses the Hathor Unleash Server and Proxy to determine if the
+   * wallet should have the Atomic Swap feature
+   *
+   * @return {boolean} The result from the unleash feature flag
+   */
+  async shouldUseAtomicSwap() {
+    try {
+      await this.client.updateContext({ userId: this.userId });
+
+      // Start polling for feature flag updates
+      await this.client.start();
+
+      // start() method will have already called the fetchToggles, so the flag should be enabled
+      this.atomicSwapEnabled = this.client.isEnabled(this.atomicSwapFlag);
+
+      return this.atomicSwapEnabled;
+    } catch (e) {
+      // If our feature flag service is unavailable, we default to the
+      // old facade
+      return false;
```

Comment:
I would add a XXX here that after the flag is released, the default must be `true`, otherwise the atomic swap would disappear from users' wallets if unleash is unavailable.